source/ncengine/graphics2/NcGraphicsImpl2.cpp

```diff
@@ -322,6 +328,7 @@ void NcGraphicsImpl2::Run()
     m_shaderBindings.GetMeshBuffer().SetBuffers(context);
 
     m_materialPassBackend.Render(context, renderState.meshRendererState.passBatches);
+    m_wireframePass.Render(context, renderState.wireframeRenderState);
```

Comment:
All of these probably belong in some sort of pass manager thing. I don't know if you've started anything like that, and it would probably just be more merge conflicts for you, so I've left it alone. Can do once you're current changes are in, if needed.